src/assets/scss/components/_oktaSigninWidget.scss

```
@@ -164,8 +164,16 @@ sign-up-modal {
       width: 50%;
     }
 
+    &[data-se="o-form-fieldset-userProfile.state"] .o-form-input-error {
+      border-radius: 0 0 0 6px;
+    }
+
     &[data-se="o-form-fieldset-userProfile.zipCode"] {
       border-left: 1px solid $okta-grey;
+
+      div.o-form-input-error {
```

All the other times we select for `.o-form-input-error`, we aren't specifying that it needs to be a `div`.

```suggestion
      .o-form-input-error {
```